pkg/bgpv1/manager/reconcilerv2/paths.go

```diff
@@ -188,3 +231,15 @@ func addPathToAFPathsMap(m AFPathsMap, fam types.Family, path *types.Path) {
 	}
 	pathsPerFamily[path.NLRI.String()] = path
 }
+
+func computePathReferences(resourcePaths ResourceAFPathsMap) PathReferencesMap {
+	pathRefs := make(PathReferencesMap)
+	for _, resAFPaths := range resourcePaths {
+		for _, afPaths := range resAFPaths {
+			for _, path := range afPaths {
+				pathRefs[path.NLRI.String()] += 1
```

Comment:
Should use pathKey from afPaths map? ( not to assume NLRI.String() is the key ).